### PR TITLE
Order stats tracking

### PR DESCRIPTION
**Pull request by gskerry**
Checkout now adding purchase stats to film db.
image
image

### CODE REVIEW

**Comment by itsaliu90** [comment]
:ok:

File: server/db/models/film.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ var filmSchema = new mongoose.Schema({
   price: {type: Number, required: true},
   photo: {type: String, default: 'http://placehold.it/200x300'},
   inventory: {type: Number, required: true},
+  purchased: {type: Number, required: true}
```
Comment (carolynyoo) [inline server/db/models/film.js]: can we add a default 0? and perhaps add this property to seed so it works?

Comment (gskerry) [inline server/db/models/film.js]: We can. Doesn't need to be a required field or present in the seed to work though. Will track purchases from app and add stats over time.

**Comment by carolynyoo** [comment]
sweet :haircut:

**Comment by gskerry** [comment]
Refactored per Gabe's advice on moving to factory. Appears to be updating db correctly.

**Comment by itsaliu90** [comment]
I checked out your branch but can't seem to see the `purchased` property in the DB after making some purchases - let's review it in the AM tomorrow.

**Comment by gskerry** [comment]
hmm. weirdness. Works in shots above.

**Comment by itsaliu90** [comment]
Okay, after much consternation, it appear to work and deserve :hamburger:. Nice!

**Pull request merged**